ex1.html

```diff
@@ -18,53 +20,57 @@
 					<br><br>
 					<input type="password" placeholder="Password" name="pass_box" class="loginboxes">
 					<br><br><br><br><br><br><br>
```

Comment:
Lots of spacing here, which is not necessarily bad, but it's coming from the HTML, which should be the keeper of _what_ data to display, not _how_ to display said data. The latter is what CSS is for. ;)